modules/install-sno-generating-the-install-iso-manually.adoc

```diff
@@ -104,21 +104,23 @@ networking: <5>
   - 172.30.0.0/16
 platform:
   none: {}
+    architecture: "<architecture>"  <7>
 bootstrapInPlace:
-  installationDisk: /dev/disk/by-id/<disk_id> <7>
-pullSecret: '<pull_secret>' <8>
+  installationDisk: /dev/disk/by-id/<disk_id> <8>
+pullSecret: '<pull_secret>' <9>
 sshKey: |
-  <ssh_key> <9>
+  <ssh_key> <10>
 ----
 <1> Add the cluster domain name.
 <2> Set the `compute` replicas to `0`. This makes the control plane node schedulable.
 <3> Set the `controlPlane` replicas to `1`. In conjunction with the previous `compute` setting, this setting ensures the cluster runs on a single node.
 <4> Set the `metadata` name to the cluster name.
 <5> Set the `networking` details. OVN-Kubernetes is the only allowed network plugin type for single-node clusters.
 <6> Set the `cidr` value to match the subnet of the {sno} cluster.
-<7> Set the path to the installation disk drive, for example, `/dev/disk/by-id/wwn-0x64cd98f04fde100024684cf3034da5c2`.
-<8> Copy the {cluster-manager-url-pull} and add the contents to this configuration setting.
-<9> Add the public SSH key from the administration host so that you can log in to the cluster after installation.
+<7> Set the architecture to `aarch64` for ARM or `x86_64` for x86 architectures.
```

Comment:
Revisited this. I'd say you can add to 120.  When the the architecture field is empty, it picks up the binary's compiled architecture.

Reply:
@prb112 Is this entry in the YAML valid here I was not 100% sure. I am adding this field to the YAML and it was not present before so I think it needs verified.

Reply:
It doesn't look like it's supported . https://github.com/openshift/installer/blob/43d0bd0b8eb90a9127bd718eec553ee53f5af166/pkg/types/none/platform.go

Reply:
Rather than modifying the install-config.yaml, I think the warning I presented in the first comment would be sufficient as a note.

Reply:
Hi @prb112 TBH i don't see how I can add this note as it would be referring to a field that is not even present in the YAML. I am still confused we state in step 2 "Set the host architecture:"

Reply:
Let's sync up and see what we can resolve. Thanks, Paul

Reply:
I looked around for similar install-config.yaml and updated to include arch fields with you guidance as well.